docker/compose-spec-l2-services.yml

```diff
@@ -199,34 +166,10 @@ services:
       linea:
         ipv4_address: 11.11.11.115
 
-  prover:
-    container_name: prover
-    hostname: prover
-    image: consensys/linea-prover:b4550ce-untested
-    platform: linux/amd64
-    # to avoid spinning up on CI for now
-    profiles: [ "l2" ]
-    environment:
-      GOMAXPROCS: 16
-      CONTROLLER_CONFIG: "/opt/linea/prover/config/controller/controller.config.toml"
-      GOMEMLIMIT: "10GiB"
-      WORKER_ID: prover-i1 # keep this prover- to mimic prod env prover-aggregation-91
-    volumes:
-      - logs:/logs
-      - ../tmp/local/:/data/
-      - ./config/prover/v2/prover-controller.config.toml:/opt/linea/prover/config/controller/controller.config.toml
-      - ./config/prover/v2/prover-execution.config.toml:/opt/linea/prover/config/executor/execution.config.toml
-      - ./config/prover/v2/prover-decompression.config.toml:/opt/linea/prover/config/executor/decompression.config.toml
-      - ./config/prover/v2/prover-aggregation.config.toml:/opt/linea/prover/config/executor/aggregation.config.toml
-      - ../config/common/traces-limits-v1.toml:/opt/linea/prover/config/executor/traces-limits-M.toml
-    networks:
-      linea:
-        ipv4_address: 11.11.11.108
-
   prover-v3: # prover compatible with the traces from zkbesu
     container_name: prover-v3
```

Comment:
We don't need distinction between prover v3 and the old prover now, but it's Ok to add it like that in case in the future we add prover-v4 alongside prover-v3